app/views/layouts/_navigation.haml

```diff
@@ -18,4 +18,8 @@
         %li.page-scroll
           %a{:href => "#about"} About
         %li.page-scroll
+          %a{:href => "#news"} News
+        %li.page-scroll
+          %a{:href => "#learnings"} Learnings
```

Comment:
Hash attribute should start with one space after the opening brace<br>Hash attribute should end with one space before the closing brace